src/NServiceBus.Core/DetectObsoleteConfigurationSettings.cs

```diff
@@ -30,26 +37,36 @@ protected internal override void Setup(FeatureConfigurationContext context)
                 throw new NotSupportedException($"The {nameof(UnicastBusConfig.DistributorDataAddress)} attribute in the {nameof(UnicastBusConfig)} configuration section is no longer supported. Remove this from the configuration section. Switch to the code API by using `{nameof(EndpointConfiguration)}.EnlistWithLegacyMSMQDistributor` instead.");
             }
 
-            var masterNodeConfig = context.Settings.GetConfigSection<MasterNodeConfig>();
+            if (unicastBusConfig?.TimeToBeReceivedOnForwardedMessages != null)
+            {
+                Logger.Warn($"The use of the {nameof(UnicastBusConfig.TimeToBeReceivedOnForwardedMessages)} attribute in the {nameof(UnicastBusConfig)} configuration section is discouraged and will be removed in the next major version.");
```

Comment:
I thought it was going to be removed completely and not replaced with code first?

> The TTBR option was a left over from the days when forwarding == audit. Now that we have a dedicated feature for auditing I don't see any use case for applying ttbr on forwarding.